easy-ui-react/src/SplitToggleCard/SplitToggleCard.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React, { useState, useId } from "react";
+import { SplitToggleCard, SplitToggleCardProps } from "./SplitToggleCard";
+import { Text } from "../Text";
+import { VerticalStack } from "../VerticalStack";
+
+type Story = StoryObj<typeof SplitToggleCard>;
+
+const meta: Meta<typeof SplitToggleCard> = {
+  title: "Components/SplitToggleCard",
+  component: SplitToggleCard,
+  args: {
+    isDisabled: false,
+    isReadOnly: false,
+  },
+  parameters: {
+    controls: {
+      exclude: ["onChange"],
+    },
+  },
+};
+
+const Template = (args: SplitToggleCardProps) => {
+  const id = useId();
+  const status =
+    args.isSelected || args.defaultSelected ? "Enabled" : "Disabled";
+  return (
+    <SplitToggleCard {...args} aria-labelledby={id}>
+      <VerticalStack gap="1">
+        <Text variant="subtitle2" color="primary.800">
+          Toggle Label
+        </Text>
+        <div>
+          <Text
+            variant="caption"
+            color={
+              args.isSelected || args.defaultSelected
+                ? "positive.600"
+                : "negative.600"
+            }
+          >
+            {status}
+          </Text>
+          <Text variant="caption" color="primary.800">
+            : AI-based Delivery Timelines for each Shipping Option at Checkout
+            for Shoppers.
+          </Text>
+        </div>
+      </VerticalStack>
+    </SplitToggleCard>
+  );
+};
+
+export default meta;
+
+export const Standard: Story = {
+  render: () => {
+    const id = useId();
+    const [selected, setSelected] = useState(false);
+    const status = selected ? "Enabled" : "Disabled";
+    return (
+      <SplitToggleCard
+        isSelected={selected}
+        onChange={setSelected}
+        aria-labelledby={id}
+      >
+        <VerticalStack gap="1">
+          <Text variant="subtitle2" color="primary.800">
+            Toggle Label
+          </Text>
+          <div>
```

Comment:
**suggestion**: i would suggest having this `<div>` (and the other story) be `<Text variant="caption">` and then the `Text` components underneath it just setting the `color`. `Text` is built to be wrapped as much as needed

as it is right now the line height is off because it's using the parent's line height

<img width="598" alt="image" src="https://github.com/user-attachments/assets/668c0226-de0f-438d-83a0-307013b448d0">

Reply:
good call, addressed